ui/lib/pki/addon/components/page/pki-configuration-details.hbs

```diff
@@ -38,14 +38,37 @@
     <h2 class="title is-4 has-bottom-margin-xs has-top-margin-xl has-border-bottom-light has-bottom-padding-s">
       Certificate Revocation List (CRL)
     </h2>
-    <InfoTableRow @label="Expiry" @value={{@crl.expiry}} />
-    <InfoTableRow @label="Auto-rebuild" @value={{if @crl.autoRebuild "On" "Off"}} />
-
+    <InfoTableRow @label="CRL building" @value={{if @crl.disable "Disabled" "Enabled"}} />
```

Comment:
thanks so much for updating this! love that it creates more consistency in the app.